src/trie/rune_util.h

```diff
@@ -0,0 +1,26 @@
+#ifndef __RUNE_UTIL_H__
+#define __RUNE_UTIL_H__
+
+#include "../dep/libnu/libnu.h"
+
+/* Internally, the trie works with 16/32 bit "Runes", i.e. fixed with unicode
+ * characters. 16 bit shuold be fine for most use cases */
+#ifdef TRIE_32BIT_RUNES
+    typedef u_int32_t rune;
+    #define TRIE_RUNE_MASK 0xffffffff
+#else // default - 16 bit runes
+    typedef u_int16_t rune;
+    #define TRIE_RUNE_MASK 0x0000ffff
```

Comment:
Nice catch. What does it do, ensure that the values aren't larger than 16 bytes?